example/storybook/stories/TrackTile/Pillar.stories.tsx

```diff
@@ -0,0 +1,164 @@
+import React from 'react';
+import { View, StyleSheet } from 'react-native';
+import { storiesOf } from '@storybook/react-native';
+import { Pillar } from 'src/components/TrackTile/PillarsTile/Pillar';
+import { action } from '@storybook/addon-actions';
+import { MockEnvironmentDecorator } from './util/MockEnvironmentDecorator';
+import {
+  Tracker,
+  TrackerValue,
+  TRACKER_PILLAR_CODE,
+  TRACKER_PILLAR_CODE_SYSTEM,
+} from 'src/components/TrackTile/services/TrackTileService';
+import { boolean } from '@storybook/addon-knobs';
+import { Anchor, CheckCircle, PlusSquare } from '@lifeomic/chromicons-native';
+import { StyleOverridesProvider } from 'src/components/TrackTile/styles';
+
+const nutritionTracker: Tracker = {
+  id: 'nutrition',
+  account: 'lifeomiclife',
```

Comment:
Because it's not in src code (and none were your initial migration PR - I definitely checked), this is a nit-pick and not at all crucial.  Going forward, if you could strip this account id out while migrating files that'd be helpful.  By searching for it, we can also ensure it's never in src code/logic.  This could be, for example, `myaccountid` or something.